vendor/k8s.io/kubernetes/pkg/kubectl/cmd/get.go

```diff
@@ -299,6 +300,11 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 		if err := printer.PrintObj(obj, out); err != nil {
 			allErrs = append(allErrs, err)
 		}
+
+		// output empty list warning when no resources to display
+		if len(infos) == 0 {
+			fmt.Fprintf(os.Stderr, "%s\n", "There are no resources to display.")
```

Comment:
I don't think we need to print the user, but the namespace seems like a good idea. Even because "user" and "logged in" are not concepts as clear in Kube as they are in Origin, it's more like the session token. Also, remember that "project" is not a concept in Kube, we need to refer to it as "namespaces". So maybe something like 

```
No resources to display in namespace "mynamespace".
No pod(s) to display in namespace "mynamespace".
```

Resource name might be harder given `all`, plural/singular etc. 

Remember that when running `get` with flags that filter the result, it may be misguiding to say "no resources" , "no pods", etc; it can give the wrong impression that there are no resources when actually just there aren't resources *that match the provided criteria*. I'd like to use some wording that makes it clear. Maybe

```
$ oc get all --selector="app=awesome"
No resources found in namespace "mynamespace" that match your criteria.
```

Ping @liggitt my Personal Assistant For Grammar Questions™.